apps/website/app/(docs)/docs/roam/pages/node-index.md

```
@@ -0,0 +1,17 @@
+author: ""
+published: true
+---
+
+- [Discourse Context](./discourse-context)
+- [Discourse Context Overlay](./discourse-context-overlay)
+- [Discourse Attributes](./discourse-attributes)
+- [Node Index](./node-index)
+
+The extension has node settings tabs for each node you define, which provides a query over all of your nodes of that type. For example, you can go to the Claim tab to see all the Claim nodes in your graph.
+
+![](/docs/roam/settings-node-index.png)
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)